### ISSUE TITLE
ID field attribute for text boxes does not works as expected

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/gateway-language-translation-suite#590]
![Screen Shot 2020-12-17 at 12.10.06 PM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/12e8ea4d-d1d9-4cc0-9674-00d9fe35390b)

id attribute for text boxes in TA/TW articles changes when text is edited in the box.
For e.g, tW article>Almighty> the header text box has id= almighty, but when the text is edited in the box, say. Almighy God, the id also changes to id= almightygod.

**Comment by elsylambert** [comment]
Found that the id remains the same now after the edits i.e id=almighty, but the element is not found by katalon anymore.

**Comment by birchamp** [comment]
@elsylambert is this still an issue?

**Comment by elsylambert** [comment]
@birchamp yes it is still an issue, but not a critical one and we have a workaround for it.
